parameterized_api_calls.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.0'
+workflow:
+  tasks:
+    task_1:
+      action: civis.import
+      input:
+        sync_type: Dbsync
+        is_outbound: False
+        source:
+          credential_id: cred1
```

Comment:
@mmatkinson can we make these numbers? `credential_id` doesn't accept a string.

Reply:
How would one do that?

Reply:
is there a way to force them to be numbers in the yml?